package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Is `package.json` meant to be included in the source distribution generated by `python setup.py sdist`?